modules/client/src/client-common/interfaces.ts

```diff
@@ -5,13 +5,14 @@ import { Contract, ContractInterface } from "@ethersproject/contracts";
 import { JsonRpcProvider } from "@ethersproject/providers";
 import { Client as IpfsClient, PinResponse } from "@aragon/sdk-ipfs";
 import { GraphQLClient } from "graphql-request";
-import { GasFeeEstimation } from "./types/common";
+import { GasFeeEstimation, SupportedNetwork } from "./types/common";
 
 export interface IClientWeb3Core {
   shiftProvider: () => void;
   getSigner: () => Signer;
   getConnectedSigner: () => Signer;
   getProvider: () => JsonRpcProvider;
+  getNetworkName: () => SupportedNetwork;
```

Comment:
Would it make sense to turn these values into an `enum`? The string values could still be the same, but people would be dealing with `SupportedNetworks.MUMBAI` instead of `maticmum`

Reply:
Yep that would make sense and will cover us in case we upgrade to ethers 6 and the names change again